models/odos/optimism/odos_optimism_trades.sql

```diff
@@ -0,0 +1,53 @@
+{{ config(
+    schema = 'odos_optimism'
+    ,alias = 'trades'
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['tx_hash', 'evt_index']
+    ,incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+WITH 
+dexs as (
+        SELECT 
+            evt_block_time as block_time, 
+            evt_block_number as block_number,
+            json_extract_scalar(outputs[1], '$.receiver') AS taker,
+            sender as maker,
+            amountsIn[1] as token_sold_amount_raw, 
+            amountsOut[1] as token_bought_amount_raw, 
+            CAST(NULL as double) as amount_usd, 
+            tokensIn[1] as token_sold_address,
+            json_extract_scalar(outputs[1], '$.tokenAddress') AS token_bought_address, 
```

Comment:
Hi @ARDev097,
please recover the changes in the last few commits.
I think the problem may be caused by this line.
As `json_extract_scalar(outputs[1], '$.tokenAddress')` may return `varchar` type,
which is not `varbinary` type.

Hence results in incompatible types.

Reply:
Done .